### DISCUSSION TITLE
[#991887] You can die in Editor Mode

### DISCUSSION THREAD

**Discussion by ihategravel2** [Mojang/minecraft-editor#20] [Bugs]
In my other bug report, 'https://github.com/Mojang/minecraft-editor/discussions/13', I reported that mobs will attack you. I want to further expand upon this and say that when you die from the resulting attack, you can't respawn.

In the video below, this is demonstrated. I die from a spider, and I cannot click the Respawn button. When I go into Crosshair Mode, my cursor is gone and I can't click it either.

Link to the video: https://twitter.com/ihategravel2/status/1633544789655167002

**Comment by minecraftchloe** [comment]
Thank you! Updated title to reflect ID for internal tracking!

**Comment by rpoerschke** [comment]
Fixed in 0.3.3.  Please reopen if you find any additional issues. Thanks!
